scripts/install_dependencies.sh

```diff
@@ -127,6 +134,30 @@ function install_dependencies() {
     pip3 install libclang parsec setuptools wheel twine --user
   fi
 
+  if [[ "${platform}" == *"Darwin"* ]]; then
+    git config --global http.postBuffer 1048576000
+
+    brew install cmake double-conversion etcd protobuf apache-arrow openmpi boost glog gflags \
+      zstd snappy lz4 openssl@1.1 libevent fmt clang autoconf go maven
+
+    export OPENSSL_ROOT_DIR="/usr/local/opt/openssl"
+    export OPENSSL_LIBRARIES="/usr/local/opt/openssl/lib"
+    export OPENSSL_SSL_LIBRARY="/usr/local/opt/openssl/lib/libssl.dylib"
+
+    # install folly
+    wget https://github.com/facebook/folly/archive/v2020.10.19.00.tar.gz -P /tmp
```

Comment:
Why not use folly formulae in brew instead build from scratch.

Reply:
brew install folly has problem to build vineyard.